lib/modules/manager/sbt/index.ts

```diff
@@ -13,11 +13,11 @@ export const url = 'https://www.scala-sbt.org';
 export const categories: Category[] = ['java'];
 
 export const defaultConfig = {
-  fileMatch: [
-    '\\.sbt$',
-    'project/[^/]*\\.scala$',
-    'project/build\\.properties$',
-    '(^|/)repositories$',
+  filePatterns: [
+    '/**/*.sbt',
+    '/project/**/*.scala',
+    '/project/build.properties',
+    '/**/repositories',
```

Comment:
```suggestion
    '**/*.sbt',
    'project/*.scala',
    'project/build.properties',
    '**/repositories',
```

Reply:
Should I first add one extra PR where I test all the default filePatterns against expected filenames so that the test would have failed in this case..?